macros/src/parser/state_machine.rs

```diff
@@ -142,12 +142,12 @@ impl parse::Parse for StateMachine {
                     return Err(parse::Error::new(
                         input.span(),
                         format!(
-                            "Unknown keyword {}. Support keywords: [\"name\", \
-                                \"transitions\", \
-                                \"temporary_context\", \
-                                \"custom_guard_error\", \
-                                \"derive_states\", \
-                                \"derive_events\"
+                            "Unknown keyword {}. Support keywords: [\"name\",
```

Comment:
Out of curiosity, how does this shown up in the console? The ` \` ending were supposed to prevent newlines from being inserted into the formatted string.

Reply:
I think it was some autoformat, but in my console it looks good though.

```
error: Unknown keyword unknown. Support keywords: ["name",
                                       "transitions",
                                       "temporary_context",
                                       "custom_guard_error",
                                       "derive_states",
                                       "derive_events",
                                       ]
```
but since it has nothing to do with this MR, I'll revert it.